cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1059,6 +1062,8 @@ def _add_head_node(self):
             indent=4,
         )
 
+        cloudformation_url = f"https://cloudformation.{self.stack.region}.{get_url_domain_suffix()}"
```

Comment:
we should define a common function here to retrieve cloudformation_url and avoid duplication.

Reply:
I agree, will address this improvement in follow up PRs.

Reply:
Addressing in this PR.